step.yml

```diff
@@ -93,3 +93,17 @@ inputs:
       - "yes"
       - "no"
 outputs:
+  - SWIFTLINT_REPORT: null
+    opts:
+      title: A Swiftlint report
+      summary: The generated Swiftlint report. May be empty if the Swiftlint succeeded with 0 errors or warnings. This is useful when you want to pass it somewhere else, for example on a Slack channel.
+      description: The generated Swiftlint report. May be empty if the Swiftlint succeeded with 0 errors or warnings. This is useful when you want to pass it somewhere else, for example on a Slack channel.
+      is_dont_change_value: true
+      category: Output
+  - SWIFTLINT_REPORT_PATH: null
+    opts:
+      title: A Swiftlint report path
+      summary: The path to the generated Swiftlint report ($BITRISE_DEPLOY_DIR by default). This is useful if you want to have this report in a separate file and do something with it, for example upload to S3.
+      description: The path to the generated Swiftlint report ($BITRISE_DEPLOY_DIR by default). This is useful if you want to have this report in a separate file and do something with it, for example upload to S3.
```

Comment:
It says that the `$BITRISE_DEPLOY_DIR` is default but you set `null`?

Reply:
Yes, I don't think it's possible to say here what will be the path, because the output file name depends on the reporter that the users selects. However, I will update the summary field to better describe this.